schema/spacewalk/upgrade/susemanager-schema-4.0.16-to-susemanager-schema-4.0.17/001-rpm.pkb.sql

```diff
@@ -0,0 +1,249 @@
+-- create schema rpm;
```

Comment:
Incorrect folder for the script!

With this the users with Uyuni installed and already at version 4.1.x would not get this SQL executed.

Use `susemanager-schema-4.1.1-to-susemanager-schema-4.1.2` (you will need to create it as I just tagged).

Reply:
Thanks. Fixed